kernel/comps/mlsdisk/src/lib.rs

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MPL-2.0
```

Comment:
The `mlsdisk` crate is neither imported by the `kernel` crate nor has any unit tests. Any particular obstacles to demonstrate that the `mlsdisk` crate can actually work?